common/Action.ts

```diff
@@ -107,10 +112,19 @@ export abstract class Action {
 	}
 
 	private async error(error: Error) {
+		console.log('::stop-commands::' + uuid());
```

Comment:
this log is intentional?

Reply:
moved it back. We use this to prevent the workflow action from rendering command/events with content in them.